### PR TITLE
Deploy portfolio using github pages

### PR DESCRIPTION
**Pull request by chiztechnology**
Hello!
After fixing issues on my portfolio I'm ready to deploy it.
Please find the code previously reviewed and the changes made in the files bellow.

Yours faithfully 🤝

### CODE REVIEW

**Comment by chiztechnology** [comment]
After merging this branch with the main branch; the web page deployed on GitHub will be refreshed then because it's deploying from the main branch.

please find the link to the online deployed version:
https://chiztechnology.github.io/Israel-Portfolio/

File: styles.css [hunk 1]
```diff
@@ -570,15 +570,15 @@ header {
...
-    width: 1440px;
+    width: 100%;
   }
 
   .main-div {
-    width: 1440px;
+    width: 100%;
   }
 
   .main-div main {
```
Comment (TadesseDev) [inline styles.css:582]: - [ ] Dear @chiztechnology, I have seen your design from the new branch with the changes, I can see you have applied change for the display greater than `768px` screen, Yet for any screen less than that it is fixed on `375px`. Please address this issue as it makes your design nonresponsive. 

## device less than `768px`
- Here we need to take the whole width of the available screen.
Capture


## device greater than `768px` (this one is good)
- Here we need to not exceed the available screen. (there should not be a scroll bar, the page should shrink to the available screen)
Capture-2

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @chiztechnology 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
I have seen your deployment from the new branch with the changes, and I have some suggestions you should work on, But you are almost there.
199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

## Highlights
 - Good job Deploying the project. ✔
 
## Required Changes ♻️
- [ ] While you did a great job deploying the project, We need to have the link to the live version on the readme file.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chiztechnology** [comment]
Hey ! thanks alot dear Tadesse for your good review. it's really helping me improving my programming skills.
Please check out the changes I made and let me know if it's time to merge with the main branch.
Faithfully 🤝 🎊

**Comment by chiztechnology** [comment]
I updated as well my readme file as requested

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @chiztechnology    
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Good job Deploying the project.  
✔️ Good Commit Messages


## Optional suggestions

- [ ] Kindly  update your README to include a link to the online version.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**